File: app/src/main/java/com/example/billeteramodulo4/Request_Money.java
package com.example.billeteramodulo4;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageButton;

import androidx.appcompat.app.AppCompatActivity;

public class Request_Money extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.request_money);

        // Abrir nueva activity

        ImageButton boton = findViewById(R.id.sendhome);

        boton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

                //ir a activity 5
                Intent i = new Intent(Request_Money.this, Send_Money.class);
                startActivity(i);
            }
        });


    }
}

File: app/src/main/java/com/example/billeteramodulo4/RequestMoney.kt
package com.example.billeteramodulo4

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import androidx.navigation.fragment.findNavController
import com.example.billeteramodulo4.databinding.FragmentRequestMoneyBinding


class RequestMoney : Fragment() {

    private lateinit var binding: FragmentRequestMoneyBinding

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View {

        binding = FragmentRequestMoneyBinding.inflate(layoutInflater)
        val root = binding.root
        // Inflate the layout for this fragment

        binding.btnenviarequestmoneypage.setOnClickListener {
            findNavController().navigate(R.id.action_requestMoney_to_homePage)
        }

        return root
    }


}